pkg/kubectl/cmd/wait/wait.go

```diff
@@ -206,9 +206,9 @@ type ResourceLocation struct {
 // UIDMap maps ResourceLocation with UID
 type UIDMap map[ResourceLocation]types.UID
 
-// WaitOptions is a set of options that allows you to wait.  This is the object reflects the runtime needs of a wait
+// Options is a set of options that allows you to wait.  This is the object reflects the runtime needs of a wait
 // command, making the logic itself easy to unit test with our existing mocks.
-type WaitOptions struct {
+type Options struct {
```

Comment:
Whoops, didn't save this comment before submitting my review :)

`WaitOptions` is used in files outside of this one... please make sure you update all references.

Reply:
I think it is not recommended to fix this golint error. There have been pending discussions going on for this change at various places - https://github.com/kubernetes/kubernetes/pull/70682#discussion_r233642556 , https://github.com/kubernetes/kubernetes/pull/78634 .

This needs to be sorted out and written in the contribution guidelines.

Reply:
@mattjmcnaughton @palnabarun  thanks for your reviews and comments :)

> We don't need to blindly follow all golint suggestions, I consider this just that. That's why I think we should leave the current naming scheme as is. -  #70682 (comment)

hmm.. I think so.